.github/CODEOWNERS

```diff
@@ -25,6 +25,7 @@
 /pytorch_lightning/distributed          @williamfalcon @tchaton @awaelchli @kaushikb11
 /pytorch_lightning/loggers              @tchaton @awaelchli @borda
 /pytorch_lightning/loggers/wandb.py     @borisdayma
+/pytorch_lightning/loggers/neptune.py   @shnela @HubertJaworski @pkasprzyk @pitercl @Raalsky @aniezurawski @kamil-kaczmarek
```

Comment:
```suggestion
```
@carmocca  suggested to do this in a separate PR, then we can merge this PR without waiting for admin to approve. (codeowners update requires admin manual merge)

Reply:
we don't have edit access to your branch, you should see a "commit suggestion" button here so please click it :) 
I will enable auto-merge

Reply:
done

Reply:
Separate PR for updating code owners: https://github.com/PyTorchLightning/pytorch-lightning/pull/9439

Reply:
Sorry, I didn't noteiced you've just created one.